frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -115,8 +168,8 @@ const Schedule = () => {
             max={
               viewState ? defaultEnd : new Date(curStart.getTime() + 7199999)
             }
-            defaultDate={new Date(2018, 0, 29)} // temp date
-            resources={resourceMap1}
+            defaultDate={new Date(2020, 10, 29)} // temp date
```

Comment:
why not just use the current date?

Reply:
I used this date to do testing (because unlike the current date, there are some rides on this date). I'm thinking about leaving it there just so we can do other testing? but if that's unnecessary I can also set to the current date

Reply:
i think you should use the current date, so the schedule functions as it's supposed to. if you wanna do testing you can easily switch it out locally as needed